manifests/n/Netstumbler/Netstumbler040/0.4.0/Netstumbler.Netstumbler040.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Netstumbler.Netstumbler040
+PackageVersion: 0.4.0
+PackageLocale: en-US
+Publisher: Netstumbler

```

Comment:
Publisher is blank in ARP, this will cause issues

Reply:
Still learning the package manifest process. Not seeing advice on how to resolve with Google.
Please advise.